node/tests/evm_zk_verify_precompile.rs

```diff
@@ -20,7 +20,7 @@ use ethers::{
 };
 
 #[cfg(not(feature = "testnet"))]
-const CHAIN_ID: u64 = 8866u64;
+const CHAIN_ID: u64 = 8886u64;
```

Comment:
So for Sydney the CHAIN_ID is 8886 but for mainnet it's 8866

Reply:
Yeah, the thing is that currently not(testnet) stands for Sydney. Testnet feature is actually Sydney. And we dont have prod codes yet